var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -55,6 +55,10 @@ class Qmcpack(CMakePackage, CudaPackage):
             description='Install with Matplotlib (long installation time)')
     variant('qe', default=True,
             description='Install with patched Quantum Espresso 6.4.1')
+    variant('afqmc', default=False,
+            description='Install with AFQMC support')
+    variant('nccl', default=False,
+            description='Include NCCL libraries for AFQMC CUDA build')
```

Comment:
So there is an `+nccl` that activates CUDA for AFQMC and `+cuda` that activates CUDA for real-space QMCPACK? Probably best to keep them independent.

Should we change the description for the `nccl` variant to:
`AFQMC CUDA build which requires NCCL libraries.`

Reply:
I looks to me from the CMake that there are two options for afqmc with cuda. One is activated by ENABLE_CUDA and then you can optionally have nccl with that. I mistook the ENABLE_CUDA for QMC_CUDA, which only seems to be for real-space. 

So I will correct this with a afqmc specific cuda option, and also the ability to include the NCCL alongside that